specification/vmware/Microsoft.AVS/models.tsp

```diff
@@ -0,0 +1,2127 @@
+import "@typespec/rest";
+import "@typespec/http";
+import "@azure-tools/typespec-azure-core";
+
+using TypeSpec.Rest;
+using TypeSpec.Http;
+using Azure.ResourceManager;
+using Azure.ResourceManager.Foundations;
+using TypeSpec.OpenAPI;
+
+namespace Microsoft.AVS;
+
+@doc("trial status")
+enum TrialStatus {
+  @doc("is available")
+  TrialAvailable,
+
+  @doc("is used")
+  TrialUsed,
+
+  @doc("is disabled")
+  TrialDisabled,
+}
+
+@doc("quote enabled")
```

Comment:
>quote [](http://example.com/codeflow?start=6&length=5)

Quota?

Reply:
Yes, fixed in d0af234a87db6c8b2ffb83da6ce4c72299a3a2ce.